graderutils/schemas/test_config_v1_1.yaml

```diff
@@ -0,0 +1,121 @@
+$schema: http://json-schema.org/draft-07/schema
```

Comment:
The idea of these is that the `test_config.yaml` should be able to state "hi, I'm written as version 1.0, so I'm valid with that" or "I'm ok with any 1.x, i.e., I don't know the minimum requirement". Thus, the system could validate the input with correct schema file or raise error that the requested version is not supported (e.g., config is written for a newer version than the library can handle. Consider that one uses too old image with a newer config example).

However, I think the schema system in grader utils doesn't handle this as well as it should. In a future, I think it should use the stuff implemented in https://github.com/apluslms/roman/tree/master/apluslms-yamlidator (or anything better).